tests/internal/src/static_analysis/well_analyzed_source.cpp

```diff
@@ -0,0 +1,3 @@
+int main(){
```

Comment:
Would you mind adding a comment here about what this is supposed to do when used in conjunction with the ``if-stmt`` clang-query file?

Reply:
Sure thing

Reply:
Thanks -- this is helpful, and was not at all obvious (to me)!

Reply:
Once I have access to my workspace again, I might try this against Blast, give a more helpful demonstration than "find if statements."